ta/crypto_perf/include/user_ta_header_defines.h

```diff
@@ -1,18 +1,19 @@
-/* SPDX-License-Identifier: BSD-2-Clause */
-/*
- * Copyright (c) 2015, Linaro Limited
- * All rights reserved.
- */
-
-#ifndef USER_TA_HEADER_DEFINES_H
-#define USER_TA_HEADER_DEFINES_H
-
-#include "ta_hash_perf.h"
-
-#define TA_UUID TA_HASH_PERF_UUID
-
-#define TA_FLAGS		(TA_FLAG_USER_MODE | TA_FLAG_EXEC_DDR)
-#define TA_STACK_SIZE		(2 * 1024)
-#define TA_DATA_SIZE		(32 * 1024)
-
-#endif
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2023, Linaro Limited

```

Comment:
I believe the copyright from 2015 should be kept since this file has only been edited.